medusa/server/api/v2/config.py

```diff
@@ -63,6 +64,7 @@ class ConfigHandler(BaseRequestHandler):
         'theme.name': StringField(app, 'THEME_NAME'),
         'backlogOverview.period': StringField(app, 'BACKLOG_PERIOD'),
         'backlogOverview.status': StringField(app, 'BACKLOG_STATUS'),
+        'layout.show.showListLayout': ListField(app, 'SHOW_LIST_ORDER'),
```

Comment:
Should this be using `showListOrder` instead of `showListLayout`?

Reply:
Ow yeah. Tnx.